lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1073,6 +1073,12 @@ def cla(self):
             self.zaxis._set_scale(self._sharez.zaxis.get_scale())
         else:
             self.zaxis._set_scale('linear')
+            try:
+                self.set_zlim(0, 1)
+            except TypeError:
+                pass
+            except AttributeError:
```

Comment:
Ok, I see what is happening. This is happening during the Axes3D constructor. It is calling the base class constructor, which calls `self.cla()`, which triggers the `Axes3D.cla()`. So, the question is, why isn't it choking first on `self.set_xlim(0, 1)`? Wouldn't it have hit that one first? I wonder what is still different between the base class calls and the Axes3D calls?

I'll see if I have some time later today to investigate further.